tests/scripts/all.sh

```diff
@@ -4637,6 +4747,7 @@ component_test_tls13_only_record_size_limit () {
 
 component_build_mingw () {
     msg "build: Windows cross build - mingw64, make (Link Library)" # ~ 30s
+    scripts/config.py unset MBEDTLS_AESNI_C # AESNI depends on cpu modifiers
```

Comment:
Disabling AESNI here is a test regression. It's apparently hiding a [product regression](https://github.com/Mbed-TLS/mbedtls/issues/8334). When tests fail, we shouldn't disable them, we should fix the bug!